packages/headless/src/controllers/commerce/search/facets/headless-search-facet-options.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '../../../../state/state-sections';
 import {CoreCommerceFacetOptions} from '../../core/facets/headless-core-commerce-facet';
 
-const facetResponseSelector = (
+export const facetResponseSelector = (
```

Comment:
We could also turn this one into a real Redux selector

Reply:
Will also do this in follow-up PR!